dev-cpp/scitokens-cpp/scitokens-cpp-1.0.1-r1.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/scitokens/scitokens-cpp"
+else
+	SRC_URI="https://github.com/scitokens/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION=" A C++ implementation of the SciTokens library with a C library interface"
+HOMEPAGE="https://scitokens.org/"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="test"
+
+DEPEND="${RDEPEND}
+	>=dev-cpp/jwt-cpp-0.6.0
```

Comment:
Is `>=` needed here? This is the minimum version we are adding to the tree in any case.

Reply:
Indeed, you are fully correct. I remember I added this due to an incompatibility with `=dev-cpp/jwt-cpp-0.4.0`, which was in GURU, but we won't carry that over so I dropped this `>=`, thanks for spotting!